site/en/blog/new-in-chrome-96/index.md

```diff
@@ -0,0 +1,222 @@
+
+Priority Hints are an experimental feature, available as an
+[origin trial][ph-ot] starting in Chrome 96, and can help optimize the Core
+Web Vitals. The `importance` attribute allows you to specify the priority
+for resource types such as CSS, fonts, scripts, images, and iframes.
+
+```html
+<!-- We don't want a high priority for this above-the-fold image -->
+<img src="/not-important.svg" importance="low">
+
+<!-- Initiate an early fetch for a resource, but de-prioritize it -->
+<link rel="preload" href="/script.js" as="script" importance="low">
+
+<script>
+  fetch('https://example.com/', {importance: 'high'}).then(data => {
+    // Trigger a high priority fetch
+  });
+</script>
+```
+
+For example, here's the Google Flights page. That background image is the
+largest contentful paint (LCP).
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/zdnbrVyWw7AMcWXVIT2g.png", alt="Screenshot of Google Flights with large background image", width="800", height="533" %}
+
+Let's see it loaded with, and without priority hints. With the priority set
+to `high` on the background image, the LCP drops from 2.6 seconds to 1.9
+seconds.
+
+{% Video src="video/0g2WvpbGRGdVs0aAPc6ObG7gkud2/MGM6YFi3IHrjRoguRr92.mp4", autoplay="true", muted="true", loop="true" %}
+
+Check out [Optimizing resource loading with Priority Hints][wd-phints] for
+all the details, how to register for the origin trial, and some great examples
+of how it can help to improve your rendering performance.
+
+## Emulate Chrome 100 in UA string {: #chrome-100 }
+
+Early next year, we'll hit Chrome 100, a **three** digit version number. Any
+code that checks version numbers, or parses the UA string, should be checked
+to make sure it handles three digits.
+
+Starting in Chrome 96, there's a new flag `#force-major-version-to-100` that
+will change the current version number to 100, so you can make sure
+everything works as expected.
+
+{% Img src="image/0g2WvpbGRGdVs0aAPc6ObG7gkud2/4drS8JxKXgtzSAxyM6WF.png", alt="Screenshot of Chrome Flags page highlighting new #force-major-version-to-100 option", width="800", height="533" %}
+
+For more details see [Force Chrome major version to 100 in the User-Agent string][dcc-cr100].
+
+## Chrome Dev Summit {: cds2021 }
+
+Chrome Dev Summit is a wrap. All of the videos and content are available
+online. Check out the [Chrome Dev Summit site][cds-site], or if you missed
+the keynote or livestream, check out the [CDS Playlist][cds-playlist] on the
+[Chrome Developers YouTube Channel][cr-dev-yt].
+
+## And more! {: #more }
+
+Of course there's plenty more.
+
+* The Back, forward cache – or `bfcache` – is now available in stable, and
+  brings Chrome in line with both Firefox and Safari.
+
+## Further reading
+
+This covers only some of the key highlights. Check the links below for
+additional changes in Chrome 96.
+
+* [What's new in Chrome DevTools (96)](/blog/new-in-devtools-96/)
+* [Chrome 96 deprecations & removals](/blog/deps-rems-96/)
```

Comment:
```suggestion
* [Chrome 96 deprecations and removals](/blog/deps-rems-96/)
```

Reply:
Fixed in 1475b8e031abe034773b744274e64066738153ad